measurements/toxicity/toxicity.py

```
@@ -0,0 +1,146 @@
+        >>> input_texts = ["she went to the library", "he is a douchebag"]
+        >>> results = toxicity.compute(predictions=input_texts)
+        >>> print(results)
+        {'toxicity': [0.0002070277841994539, 0.856372594833374]}
+
+    Example 2 (returns ratio of toxic sentences):
+        >>> toxicity = evaluate.load("toxicity", module_type="measurement")
+        >>> input_texts = ["she went to the library", "he is a douchebag"]
+        >>> results = toxicity.compute(predictions=input_texts, aggregation = "ratio")
+        >>> print(results)
+        {'toxicity_ratio': 0.5}
+
+    Example 3 (returns the maximum toxicity score):
+
+        >>> toxicity = evaluate.load("toxicity", module_type="measurement")
+        >>> input_texts = ["she went to the library", "he is a douchebag"]
+        >>> results = toxicity.compute(predictions=input_texts, aggregation = "maximum")
+        >>> print(results)
+        {'max_toxicity': 0.856372594833374}
+
+    Example 4 (uses a custom model):
+
+        >>> toxicity = evaluate.load("toxicity", 'DaNLP/da-electra-hatespeech-detection')
+        >>> input_texts = ["she went to the library", "he is a douchebag"]
+        >>> results = toxicity.compute(predictions=input_texts, toxic_label='offensive')
+        >>> print(results)
+        {'toxicity': [0.017554517835378647, 0.020320769399404526]}
+"""
+
+
+def toxicity(preds, toxic_classifier, toxic_label):
+    toxic_scores = []
+    if toxic_label in toxic_classifier.model.config.id2label.values():
+        for pred in preds:
+            pred_toxic = toxic_classifier(str(pred))
+            hate_toxic = [r["score"] for r in pred_toxic if r["label"] == toxic_label][0]
+            toxic_scores.append(hate_toxic)
+    else:
+        logger.warning(
+            "The toxic label that you specified is not part of the model labels. Run `model.config.id2label` to see what labels your model outputs."
+        )
```

I think this should throw an error. Instead of "toxic label" I would use `toxic_label` so it is clearer which kwarg it is.

---

Also if you do `if toxic_label not in ...` you can directly raise the error and unindent the code block. Makes it a bit more readable.

---

done!